### PR TITLE
Change Android URLs to Linode bucket, old HiFi one is gone

### PR DESCRIPTION
**Pull request by daleglass**
This is for fixing Android builds. It needs testing.

Also the URL will probably change to a vircadia.com one (waiting on Kalila for that)

### CODE REVIEW

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR466/Vircadia-Alpha-PR466-7778c9e_d66c8cc2.dmg

**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR466/Vircadia-Alpha-PR466-7778c9e.tar_d66c8cc2.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR466/Vircadia-Alpha-PR466-7778c9e.tar_d66c8cc2.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR466/Vircadia-Alpha-PR466-7778c9e_d66c8cc2.sh

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR466/Vircadia-Alpha-PR466-7778c9e_d66c8cc2.exe

**Comment by kasenvr** [comment]
Got the new URL online, here is such an example:

https://content.vircadia.com/eu-c-1/vircadia-public/dependencies/android/qt-5.11.1_linux_armv8-libcpp_openssl_patched.tgz

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR466/Vircadia-Alpha-PR466-ec12213_cb7ab630.dmg

**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR466/Vircadia-Alpha-PR466-ec12213.tar_cb7ab630.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR466/Vircadia-Alpha-PR466-ec12213_cb7ab630.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR466/Vircadia-Alpha-PR466-ec12213.tar_cb7ab630.gz

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR466/Vircadia-Alpha-PR466-ec12213_cb7ab630.exe

File: hifi_android.py [hunk 1]
```diff
@@ -10,87 +10,74 @@
...
     'tbb': {
         'file': 'tbb-2018_U1_armv8_libcpp.tgz',
-        'versionId': 'mrRbWnv4O4evcM1quRH43RJqimlRtaKB',
         'checksum': '20768f298f53b195e71b414b0ae240c4',
         'sharedLibFolder': 'lib/release',
         'includeLibs': ['libtbb.so', 'libtbbmalloc.so'],
     },
     'hifiAC': {
-        'baseUrl': 'http://s3.amazonaws.com/hifi-public/dependencies/',
+        'baseUrl': 'https://content.vircadia.com/eu-c-1/vircadia-public/dependencies/',
```
Comment (ctrlaltdavid) [inline hifi_android.py:71]: Why is this package in a location different to all the other packages?

Comment (daleglass) [reply hifi_android.py:71]: That's just where it was in the original hifi S3 bucket

Comment (ctrlaltdavid) [reply hifi_android.py:71]: Perhaps the package should be moved to make things tidier?

**Pull request merged**